src/main/kotlin/screeps/utils/Options.kt

```diff
@@ -1,7 +1,73 @@
 package screeps.utils
 
 import screeps.api.Game
+import screeps.api.LineStyleConstant
+import screeps.api.RoomVisual
+import screeps.api.TextAlignConstant
 
 fun routeOptions(routeCallback: (roomName: String, fromRoomName: String) -> Double): Game.Map.RouteOptions = jsObject {
     this.routeCallback = routeCallback
 }
+
+// RoomVisual style options
```

Comment:
This comment already hints at the fact that the functions below are only used with RoomVisual.
Please put them into the file screeps.api.RoomVisual.kt

As discussed in https://github.com/exaV/screeps-kotlin-types/pull/11 we should put things  that belong together into the same file.

Reply:
I am aware that there are already some deviations from this throughout the api (e.g. the file Utils.kt), we will fix  those as before 1.0 just not in this PR.